test/traits.jl

```diff
@@ -698,12 +718,4 @@ end
 
     @test_throws MethodError oneunit(RGB{N0f8})
     @test_throws MethodError zero(RGB{N0f8})
-
-    for T in (N0f8, Float32)
-        g = Gray{T}(0.8)
-        @test oneunit(g) == oneunit(T) == Gray(oneunit(T))
-        @test zero(g) == zero(T) == Gray(zero(T))
-    end
-
-    @test_broken one(Gray{Float32}) * g == g * one(Gray{Float32}) == g
```

Comment:
ColorVectorSpaces seems already fixes this, so it not a valid test anymore.

Reply:
The test means `one(Gray)` should return `1`. Do not remove it now.

Reply:
Neither `Gray{N0f8}(1) * Gray{N0f8}(0.8)` nor `1 * Gray{N0f8}(0.8` are valid without ColorVectorSpaces...

And `Gray{N0f8} == 1` is true

> The test means one(Gray) should return 1

I'll add one more test for this.

Reply:
> Neither Gray{N0f8}(1) * Gray{N0f8}(0.8) nor 1 * Gray{N0f8}(0.8 are valid without ColorVectorSpaces...

You are right. However, I think the broken test makes sense to indicate that "this is broken" and "one is an identity element".